internal/service/sagemaker/endpoint_configuration.go

```diff
@@ -227,6 +227,37 @@ func ResourceEndpointConfiguration() *schema.Resource {
 							ForceNew:     true,
 							ValidateFunc: validation.StringInSlice(sagemaker.ProductionVariantAcceleratorType_Values(), false),
 						},
+						"container_startup_health_check_timeout_in_seconds": {
+							Type:         schema.TypeInt,
+							Optional:     true,
+							ForceNew:     true,
+							ValidateFunc: validation.IntBetween(60, 3600),
+						},
+						"core_dump_config": {
```

Comment:
tricky to test so im leaving it out i did basic validation that this is not breaking something else